rest/src/test/resources/application.yml

```diff
@@ -0,0 +1,3 @@
+spring:
+  config:
+    import: classpath:mongodb/mongodb.yml, classpath:wordnik/wordnik.yml, classpath:nats/nats.yml
```

Comment:
it's nice you got familiar with import, but now your configuration looks much more complicated